projects/fsastorefrontstyles/scss/components/cms/_enriched-responsive-banner.scss

```diff
@@ -3,8 +3,80 @@
     position: relative;
     width: 100%;
   }
+  .enriched-banner-text {
+    background-color: rgba(255, 255, 255, 0.7);
+    position: absolute;
+  }
+  .enriched-banner-text-strip {
+    display: flex;
+    flex-flow: column;
+    width: 100%;
+  }
+  .enriched-banner-title {
+    font-weight: var(--cx-font-weight-semi);
+  }
+  .box-title {
+    font-size: var(--cx-h1-font-size);
```

Comment:
Maybe use clamp for font-size so we don't have to use media queries.

Reply:
Didn't manage to use clamp here. There are 3 different font sizes and for tablets screen size is bigger than for desktop...

Reply:
`clamp(60%, var(--cx-h1-font-size), 4vw)`?

Reply:
For me it's ok, but it isn't how it was. On the tablet screens font-size was bigger than on the desktop. With your suggestion size is same on the both screens.

Reply:
Looks good, so I would refactor it like this.